backend/app/controllers/spree/admin/stock_locations_controller.rb

```diff
@@ -7,9 +7,10 @@ class StockLocationsController < ResourceController
 
       def set_country
         @stock_location.country = Spree::Country.default
-      rescue ActiveRecord::RecordNotFound
-        flash[:error] = Spree.t(:stock_locations_need_a_default_country)
-        redirect_to admin_stock_locations_path
+        unless @stock_location.country
```

Comment:
@dalizard She has also changed the default method so now it won't raise an exception